include/sensor_fusion.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * \brief Function Prototype Library to implement Sensor Fusion Algorithm
+ * \author jaspalsingh@cmail.carleton.ca
+ * \version 1.0
+ * \date 2019-12-01
+ */
+
+#ifndef SENSOR_FUSION_H
+#define SENSOR_FUSION_H
```

Comment:
Standard header guard is as follows:
```
#ifndef __MY_HEADER__
#define __MY_HEADER__
```
This is listed as convention 13 in the announcement on culearn.

Reply:
Fixed in recent commits to dev branch